gradle.properties

```diff
@@ -0,0 +1 @@
+junitVersion=5.2
```

Comment:
This is confusing because it sounds like we're building against 5.2. Calling it `junitMinorVersion` and adding a comment showing how the variable is used should fix that.